tests/test_georaster.py

```diff
@@ -511,7 +511,7 @@ def test_from_array(self):
         # -> most tests already performed in test_copy, no need for more
         img = gr.Raster(datasets.get_path('landsat_B4'))
         out_img = gr.Raster.from_array(img.data, img.transform, img.crs, nodata=img.nodata)
-        assert np.array_equal(out_img.data, img.data)
+        assert out_img == img
```

Comment:
"The author of this commit finds the most beautifully prepared syntax and uses it in textbook fashion. 5/5 raspberries"  - Mannerfelt Times 2021